bib/east.bib

```diff
@@ -10121,7 +9984,7 @@ @Book{east:5384
   date       = {0800~},
   title      = {rigs pa'i thigs pa'i don bsdus pa},
   language   = {bo},
-  series     = {snar thang 3724 zhe 115a–116a; sde dge 4233 we 99b5–100b3; co ne we 103a2–103b7; {Peking} 5732 zhe 122b6–123b8},
+  series     = {snar thang 3724 zhe 115a–116a; sde dge 4233 we 99b5-100b3; co ne we 103a2-103b7; {Peking} 5732 zhe 112b6-123b8},
```

Comment:
The Peking folio/line number here seems wrong.  A typo?  122b6 seems correct, the text is only one folio long in the other editions.

Reply:
Yes, this is simply a typo, I think. The correct folio number is {Peking } 5732 zhe 122b6-123b8.